ApplyToBecomeInternal/ApplyToBecomeInternal.Tests/Pages/ProjectList/ProjectListFilteringIntegrationTests.cs

```diff
@@ -0,0 +1,92 @@
+﻿using AngleSharp.Dom;
+using AngleSharp.Html.Dom;
+using ApplyToBecome.Data.Models;
+using ApplyToBecomeInternal.Models;
+using ApplyToBecomeInternal.Tests.Extensions;
+using FluentAssertions;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace ApplyToBecomeInternal.Tests.Pages.ProjectList
+{
+	public class ProjectListFilteringIntegrationTests : BaseIntegrationTests, IAsyncLifetime
+	{
+		private int _recordCount;
+		private IList<AcademyConversionProject> _projects;
+
+		public ProjectListFilteringIntegrationTests(IntegrationTestingWebApplicationFactory factory) : base(factory)
+		{
+		}
+
+		private IHtmlElement FilterBanner => Document.QuerySelector<IHtmlElement>(Select.ProjectList.Filter.Banner.CypressSelector());
+		private IHtmlElement FilterCount => Document.QuerySelector<IHtmlElement>(Select.ProjectList.Filter.Count.CypressSelector());
+		private IHtmlElement FilterOptions => Document.QuerySelector<IHtmlElement>(Select.ProjectList.Filter.Options.CypressSelector());
+		private IEnumerable<IHtmlInputElement> FilterStatuses => Document.QuerySelectorAll<IHtmlInputElement>(Select.ProjectList.Filter.Status().CypressSelector("^="));
+		private IHtmlButtonElement FilterApply => Document.QuerySelector<IHtmlButtonElement>(Select.ProjectList.Filter.Apply.CypressSelector());
+
+		public async Task InitializeAsync()
+		{
+			_recordCount = 20;
+			_projects = AddGetProjects(recordCount: _recordCount).ToList();
+
+			await OpenUrlAsync("/project-list");
+
+			FilterStatuses.Should().NotBeEmpty();
+		}
+
+		public Task DisposeAsync()
+		{
+			return Task.CompletedTask;
```

Comment:
Can you just remove the DisposeAsync in this instance?

Reply:
No - it's part of the interface.